### PR TITLE
[dhctl] Preflight check exist embedded containerd 

### PR DESCRIPTION
**Pull request by aleksey-su**
## Description
Preflight check exist embedded containerd

## Why do we need it, and what problem does it solve?
#7960

## Checklist
- [ ] The code is covered by unit tests.
- [x] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: dhctl
type: feature
summary: Preflight check exist embedded containerd.
impact_level: default
```

### CODE REVIEW

**Comment by aleksey-su** [comment]
image

**Comment by aleksey-su** [comment]
image

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @aleksey-su. I've started the workflow for label 'e2e/run/openstack'...

File: candi/bashible/common-steps/node-group/031_install_containerd.sh.tpl [hunk 1]
```diff
@@ -39,5 +39,12 @@ post-install() {
   fi
 }
 
+if [[ "$FIRST_BASHIBLE_RUN" == "yes" ]]; then
+  if command -v containerd >/dev/null 2>&1; then
+    bb-log-error "containerd exist on servers $HOSTNAME. Deckhouse not support working with embedded containerd. Please uninstall containerd and try again"
```
Comment (mvasl) [inline candi/bashible/common-steps/node-group/031_install_containerd.sh.tpl:44]: "containerd is detected on $HOSTNAME. Deckhouse does not support pre-provisioned containerd installations. Please uninstall containerd and try again."

Comment (mvasl) [inline candi/bashible/common-steps/node-group/031_install_containerd.sh.tpl:44]: "containerd is detected on $HOSTNAME. Deckhouse does not support pre-provisioned containerd installations. Please uninstall containerd and try again."

**Review by mvasl — CHANGES_REQUESTED** [review]

File: candi/bashible/common-steps/node-group/031_install_containerd.sh.tpl (hunk 1 above)
Comment (RomanenkoDenys) [reply candi/bashible/common-steps/node-group/031_install_containerd.sh.tpl:44]: Please move it to the one of first steps. We dont need to pass to 31 step to check if containerd is exists on node

Comment (RomanenkoDenys) [reply candi/bashible/common-steps/node-group/031_install_containerd.sh.tpl:44]: 000_preflight_checks.sh

Comment (RomanenkoDenys) [reply candi/bashible/common-steps/node-group/031_install_containerd.sh.tpl:44]: please check another steps and move them to this step

**Review by RomanenkoDenys — CHANGES_REQUESTED** [review]

**Review by name212 — CHANGES_REQUESTED** [review]
agree with Denis

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @aleksey-su. I've started the workflow for label 'e2e/run/openstack'...

**Review by name212 — APPROVED** [review]

**Review by RomanenkoDenys — APPROVED** [review]

**Pull request merged**